ports/javascript/README.md

```diff
@@ -7,16 +7,16 @@ Dependencies
 ------------
 
 Building micropython.js bears the same requirements as the standard MicroPython
-ports with the addition of Emscripten (and uglify-js for the minified file). 
+ports with the addition of Emscripten (and uglify-js for the minified file).
```

Comment:
Usually, unrelated changes like fixing things like this whitespace would be done in separate commits, to keep changes self contained and easier to review/understand.